test/unit/custom_maps.spec.ts

```diff
@@ -600,7 +606,7 @@ describe("Interposing on a map", () => {
                 ]
             };
 
-            executeTestSuiteInternal("foo.json", cfg, version);
+            executeTestSuiteInternal(cfg, version);
```

Comment:
Does this need an await?

Reply:
Added await in [d19be01](https://github.com/ConsenSys/scribble/pull/143/commits/d19be011aa01087d036d8c863e32aa787b0afff4)